### PR TITLE
:sparkles: modify tested component inner state

### PR DESCRIPTION
**Pull request by fabienjuif**
#59 

TODO : 
 - [x] UI
 - [x] Tests

### CODE REVIEW

**Comment by fabienjuif** [comment]
@bpetetot A first review is welcomed before I start UI and tests :)

File: src/gui/redux/store.js [hunk 1]
```diff
@@ -21,6 +22,7 @@ const store = createStore(
     }),
     model,
     router: router.reducer,
+    state,
```
Comment (bpetetot) [inline src/gui/redux/store.js]: reducer `model` contains component `props`.
reducer `state` contains component `state`.

Should we combine them in one `component.props` and `component.state`

**Review by bpetetot — DISMISSED** [review]

**Comment by fabienjuif** [comment]
@bpetetot new review is welcomed, I think this ready to be merged

**Review by bpetetot — APPROVED** [review]
👍

**Pull request merged**